tensorflow/compiler/xla/tools/multihost_hlo_runner/BUILD

```diff
@@ -27,7 +26,7 @@ build_test(
     ],
 )
 
-xla_cc_binary(
```

Comment:
We had changed these because the denorm fix needed some defines from the TF config.proto. Some tests may fail..

Reply:
Build failed on rocm platform with current setting.

Reply:
It's built nightly as is here:
http://ml-ci.amd.com:21096/job/tensorflow/job/release-rocmfork-r213-rocm-enhanced/job/release-build-whl/

Reply:
It's not a module in tensorflow package, it's a binary for run hlo. What's test script? Such as bazel test xxx.

Reply:
oh - I read this backward - you are switching it to tf_cc_binary like a number of the other places we had to change.
This shouldn't be necessary in 2.15 and beyond.

Reply:
Yes, it's for tf2.13 and I submited it to r2.13 branch

Reply:
> It's not a module in tensorflow package, it's a binary for running hlo. What's test script? Such as bazel test xxx.

The CI will test it.